gef-ui/server/controllers/about-facility/index.js

```diff
@@ -179,10 +179,14 @@ const validateAndUpdateAboutFacility = async (req, res) => {
       return res.redirect(`/gef/application-details/${dealId}`);
     }
 
-    if (isTrueSet(isUsingFacilityEndDate)) {
+    if (isUsingFacilityEndDate === 'true') {
       // TODO: DTFS2-7161 - Implement page to submit facilityEndDate
     }
 
+    if (isUsingFacilityEndDate === 'false') {
+      return res.redirect(`/gef/application-details/${dealId}/facilities/${facilityId}/bank-review-date`);
+    }
```

Comment:
Have done so, two thoughts:
- Think it makes sense not to handle this error because the value should never fail this check
- Still need to use the string value for error handling and rerendering the page, I never like calling something `isUsingFacilityEndDateString` but i think its the best option